google-account-plugin/src/com/google/gct/stats/GoogleUsageTracker.java

```diff
@@ -71,27 +71,26 @@ public void trackEvent(@NotNull String eventCategory,
                            @NotNull String eventAction,
                            @Nullable String eventLabel,
                            @Nullable Integer eventValue) {
-        if (!ApplicationManager.getApplication().isUnitTestMode()) {
-            if (UsageTrackerManager.getInstance().isTrackingEnabled()) {
-                ArrayList postData = Lists.newArrayList(analyticsBaseData);
-                postData.add(new BasicNameValuePair("tid", analyticsId));
-                postData.add(new BasicNameValuePair(INTELLIJ_EDITION, PlatformUtils.isCommunityEdition() ? "community" : "ultimate"));
-                postData.add(new BasicNameValuePair("ec", eventCategory));
-                postData.add(new BasicNameValuePair("ea", eventAction));
-                if (!Strings.isNullOrEmpty(eventLabel)) {
-                    postData.add(new BasicNameValuePair("el", eventLabel));
-                }
+        if ((!ApplicationManager.getApplication().isUnitTestMode()) &&
```

Comment:
I'm not sure this one pulls its weight. It seems marginally harder to read. Maybe of you removed the extra parentheses?